src/detectors/BEMC/ProtoCluster_factory_EcalBarrelTruthProtoClusters.h

```diff
@@ -29,6 +29,8 @@ class ProtoCluster_factory_EcalBarrelTruthProtoClusters : public JFactoryT<edm4e
         m_inputHit_tag="EcalBarrelRecHits";
         m_inputMCHit_tag="EcalBarrelHits";
 
+        app->SetDefaultParameter("BEMC:EcalBarrelTruthProtoClusters:inputHit_tag", m_inputHit_tag, "Name of input collection to use");
```

Comment:
missing m_inputMCHit_tag exposure

Reply:
Similar to above. I think this is only used to access ID info from the truth level object so was not exposed on purpose.